### ISSUE TITLE
NVDA OCR in windows 10 fails on this website

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8299]
### Steps to reproduce:
1. Open this website
https://www.rewe.de/marktseite/koeln/1940053/rewe-city-hoeninger-weg-185/?market-flyer=active
2. You will see the Rewe Prospekt
3. Start NVDA in browse mode and go with the down arrow key until you hear "vorherige Seite"
4. Then, press up arrow once and press NVDA+r to run OCR recognition
5. You will notice that NVDA does not recognize much content
6. If NVDA recognizes something at all, it is not possible to navigate with the arrow keys through the text.

### Expected behavior:
OCR recognition should be able to recognize the text in the prospect and text navigatoin should work properly.

### Actual behavior:
There is no way to navigate through recognized content, if any.

### System configuration:
NVDA version:
NVDA 2018.1.1

NVDA Installed or portable:
Installed

Windows version:
Windows 10.0.17134

Name and version of other software in use when reproducing the issue:
Google Crome 66.0, Mozilla Firefox 60.0.1

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No

**Comment by ehollig** [comment]
@Adriani90 is this still an issue? I think the website may have changed.

**Comment by Adriani90** [comment]
Oh thanks for reminding me. Yes the website seems redesigned. This works now just fine without using OCR as well. Closing as works for me.
